docs/src/tutorial.rst

```diff
@@ -271,11 +271,13 @@ values:
                 "palette": "cividis",
                 // settings related to the markers sizes
                 "size": {
+                    // scaling factor for the axis, between 1 and 100
+                    "factor": 55,
+                    // mode to scale the markers with respect to the properties
```

Comment:
Could you add a few words describing what each option does? constant, inverse and linear might not be 100% evident.

side note, in French inverse means 1/x, not -x. Is it the same in English? If so, I would propose `reverse` for this mode

Reply:
Another non-trivial fix: I added a "reverse size" checkbox so that any size-scaling mode can be reversed